src/components/MediaDetails/MediaDetailsAudio.js

```diff
@@ -1,14 +1,33 @@
 import React from 'react'
 import PropTypes from 'prop-types'
+import { Link, useParams } from 'react-router-dom'
 
 // FPCC
 import getIcon from 'common/utils/getIcon'
 import AudioNative from 'components/AudioNative'
 import { isDisplayablePropMedia } from 'common/utils/mediaHelpers'
 
 function MediaDetailsAudio({ file }) {
+  const { sitename } = useParams()
   return (
     <div id="MediaDetailsAudio" className="mpb-16 space-y-6 sticky top-0">
+      <div className="flex justify-center space-x-2">
+        <a href={file?.downloadLink} className="flex-1 btn-outlined">
+          <div className="flex w-full h-full items-center justify-center">
+            {getIcon('Download', 'btn-icon mr-2 w-5 h-5')} Download
```

Comment:
There shouldn't be any adverse effects, but `w-5 h-5` is redundant here as it is already set in `btn-icon`.